pkg/v14/kubeconfig/desired_test.go

```diff
@@ -165,3 +166,11 @@ func Test_Resource_GetDesiredState(t *testing.T) {
 		})
 	}
 }
+
+func transform(obj interface{}) (v1alpha1.ClusterGuestConfig, bool, error) {
+	customConfig, ok := obj.(*v1alpha1.AWSClusterConfig)
+	if !ok {
+		return v1alpha1.ClusterGuestConfig{}, false, microerror.Mask(wrongTypeError)
+	}
+	return customConfig.Spec.Guest.ClusterGuestConfig, false, nil
```

Comment:
It looks like the boolean return parameter is pretty useless. We should remove it.